tarsier/tag_utils.ts

```diff
@@ -487,6 +487,10 @@ window.hideNonTagElements = () => {
   });
 };
 
+function transformXPath(xpath: string): string {
+    return xpath.replace(/\/(\w+):(\w+)/g, '/*[name()="$1:$2"]');
+}
+
```

Comment:
function naming is a bit unclear

Maybe change to 
```suggestion
function fixNamespaces(xpath: string): string {
   // Comment describing the issue, what a name space is, and why this fix works
    return xpath.replace(/\/(\w+):(\w+ )/g, '/*[name()="$1:$2"]');
}

```